src/wp-admin/includes/plugin-install.php

```diff
@@ -234,6 +245,45 @@ function plugins_api( $action, $args = array() ) {
 	return apply_filters( 'plugins_api_result', $res, $action, $args );
 }
 
+/**
+ * Sets default false values for specific fields in the $args object.
+ *
+ * This function ensures that the $args->fields array is an array and loops through the fields
+ * with default false values. If a field is not already set in $args->fields, it is set to false.
+ *
+ * @param object $args The arguments object.
+ * @return object The modified $args object with default false values for specific fields.
+ */
+function set_default_false_fields( $args ) {
```

Comment:
Let's convert this to a private function instead as this has no reason to be a public one. :)

```suggestion
/**
 * Sets default false values for specific fields in the $args object.
 *
 * This function ensures that the $args->fields array is an array and loops through the fields
 * with default false values. If a field is not already set in $args->fields, it is set to false.
 *
 * @internal
 *
 * @since 6.6.0
 * @access private
 * 
 * @see https://core.trac.wordpress.org/ticket/60783
 * 
 * @param object $args The arguments object.
 * @return object The modified $args object with default false values for specific fields.
 */
function _set_default_false_fields( $args ) {
```

Reply:
@Rajinsharwar We could have used Private, but functions are not used inside of the class in this file.